kinetic/distribution.yaml

```diff
@@ -17513,6 +17512,16 @@ repositories:
       url: https://github.com/ros/xacro.git
       version: kinetic-devel
     status: developed
+  xarm:
+    doc:
+      type: git
+      url: https://github.com/xArm-Developer/xarm_ros.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/xArm-Developer/xarm_ros.git
```

Comment:
At least some of the packages in the source repository here have TODO listed as the license in the package.xml.  Please fix those up so that all of the packages declare their licenses.

Reply:
Noted, license upgraded. Thanks.